### PR TITLE
Update to use external prize pool registry

### PR DESCRIPTION
**Pull request by aodhgan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# pooltogether-operations-contracts
+# PoolTogether PrizePool Upkeep
 
 [![Coverage Status](https://coveralls.io/repos/github/pooltogether/pooltogether-operations-contracts/badge.svg?branch=main)](https://coveralls.io/github/pooltogether/pooltogether-operations-contracts?branch=main)
```
Comment (asselstine) [inline README.md:3]: + github workflow badge

File: contracts/PrizeStrategyUpkeep.sol [hunk 2]
```diff
@@ -10,17 +10,19 @@ import "./interfaces/PrizePoolRegistryInterface.sol";
...
 
     using SafeAwardable for address;
 
-    address public prizePoolRegistry;
+    AddressRegistry public prizePoolRegistry;
 
     uint public upkeepBatchSize;
     
-    constructor(address _prizePoolRegistry, uint256 _upkeepBatchSize) public {
+    constructor(AddressRegistry _prizePoolRegistry, uint256 _upkeepBatchSize) public {
```
Comment (asselstine) [inline contracts/PrizeStrategyUpkeep.sol:25]: After running the contract on Rinkeby, we quickly hit 1m gas usage for 5 pools.  Let's make `upkeepBatchSize` configurable by the owner.

**Review by asselstine — COMMENTED** [review]
Looking good!  Just a few things

File: contracts/PrizeStrategyUpkeep.sol (hunk 2 above)
Comment (aodhgan) [reply contracts/PrizeStrategyUpkeep.sol]: Done had to make the contract Ownable)

**Pull request merged**